.gitignore

```diff
@@ -34,3 +34,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# wrangler
+.wrangler
```

Comment:
The template we give people should ignore `.wrangler`, otherwise you end up committing the temp state of local dev to git history accidentally.